src/kOS/Binding/FlightStats.cs

```diff
@@ -12,15 +12,17 @@ public class FlightStats : Binding
     {
         public override void AddTo(SharedObjects shared)
         {
-            shared.BindingMgr.AddGetter("ALT_APOAPSIS", () => shared.Vessel.orbit.ApA);
-            shared.BindingMgr.AddGetter("ALT_PERIAPSIS", () => shared.Vessel.orbit.PeA);
-            shared.BindingMgr.AddGetter("ALT_RADAR", () => Convert.ToDouble(shared.Vessel.heightFromTerrain > 0 ? Mathf.Min(shared.Vessel.heightFromTerrain, (float)shared.Vessel.altitude) : (float)shared.Vessel.altitude));
+            shared.BindingMgr.AddGetter("ALT", () => new Alt(shared));
+            shared.BindingMgr.AddGetter("ALT_APOAPSIS", () => (new Alt(shared)).GetApoapais());
```

Comment:
I was operating under the mandate to just refactor to mimic the existing behavior in a new way.  So I did.  If you don't mind a 'breaking' message, then yeah we can get rid of them.